builder

```diff
@@ -127,6 +127,17 @@ if [[ "$MONOREPO" == "true" ]]; then
       fi
     done
   fi
+
+  if [[ -n "$APP_BUILDER" ]]; then
```

Comment:
not only for mono repos

Reply:
what is APP_BUILDER?

Reply:
I guess for now, we can leave for just mono repos - to simplify

Reply:
Couple things:
1. `APP_BUILDER` is just the location of the `docker-app` executable you would pass in to trigger that you wanted to use it - since this repo doesn't hold or know about that - it would just "consume" it.
2. I think there is some complexity growing inside here that maybe warrants adding some functions to organize some things. I Started with mono-repo's since their structure goes along with dockerapp deloys more than individual apps (to start).